cli/pkg/kctrl/cmd/app/init/fetch.go

```diff
@@ -49,7 +49,11 @@ func (f *FetchStep) Interact() error {
 	}
 	isHelmTemplateExistInPreviousOption := f.helmTemplateExistInAppBuild()
 	previousFetchOptionSelected := GetFetchOptionFromVendir(vendirConfig, isHelmTemplateExistInPreviousOption)
-
+	if previousFetchOptionSelected == "MultipleFetchOptions" {
```

Comment:
Should this not be a constant defined someplace?

Reply:
As it is being used here only, hence  not a constant.